### ISSUE TITLE
Is OGG supported by SoundFile()?

### ISSUE THREAD

**Issue by q2dg** [processing/processing-sound-archive#75]
If it isn't, this issue would act as a low-priority request.
If it is, description in https://processing.org/reference/libraries/sound/SoundFile.html should be updated
Thanks!

**Comment by MikeTheShadow** [comment]
Yes it is. You can try it yourself I used this in the preload function 
var mp3file = "musicfiles/test.ogg";
sound = loadSound(mp3file);

**Comment by q2dg** [comment]
Ok, thanks. So...as I said, https://processing.org/reference/libraries/sound/SoundFile.html should be updated accordingly.
